torch/serialization.py

```diff
@@ -338,7 +338,13 @@ def save(obj, f, pickle_module=pickle, pickle_protocol=DEFAULT_PROTOCOL, _use_ne
            containing a file name
         pickle_module: module used for pickling metadata and objects
         pickle_protocol: can be specified to override the default protocol
-
+        _use_new_zipfile_serialization: saves the object as a zip file rather than a binary file
```

Comment:
This flag looks like it should not be exposed to the public because of the leading underscore.

@eellison If I remember correctly, this is a TorchScript thing. Do you know what this flag is and if we should document it?

Reply:
hey don't really know about it, maybe @zdevito or @voznesenskym would know

Reply:
Hey! qq, when you say "to the public", do you mean as part of the API? Or do you mean outside of fbcode?

Reply:
I mean as a part of the API. Generally as a "pytorch user", when I see something with underscores prepended to it, that means that it is not part of the public API

Reply:
As I understand it, the purpose of this is for a migration. Having it as part of the API for now seems like a relatively simple way to denote which serialization you want to use. I am super open to hearing alternative ways of implementing the same functionality though!